UIKitHomework/UIKitHomework/Controllers/BucketViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  BucketViewController.swift
+//  UIKitHomework
+//
+//  Created by Vlad on 27.09.2022.
+//
+
+import UIKit
+
+/// Bucket View controller. All items in orders here and final price.
+final class BucketViewController: UIViewController {
+
+    // MARK: - UIProperties
+
+    private lazy var paymentButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "applelogo"), for: .normal)
+        button.tintColor = .white
+        button.backgroundColor = .black
+        button.layer.cornerRadius = 5
+        button.setTitle("Pay", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.titleLabel?.font = UIFont.boldSystemFont(ofSize: 18)
+        button.frame = CGRect(x: 20, y: 750, width: 360, height: 50)
+        button.addTarget(self, action: #selector(paymentButtonAction), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var cardPaymentSwitch = {
+        let switcher = UISwitch()
+        switcher.isOn = true
+        switcher.frame = CGRect(x: 300, y: 650, width: 50, height: 50)
+        switcher.addTarget(self, action: #selector(changePaymentAction), for: .valueChanged)
+        return switcher
+    }()
+
+    private lazy var cashPaymentSwitch = {
+        let switcher = UISwitch()
+        switcher.isOn = false
+        switcher.frame = CGRect(x: 300, y: 700, width: 50, height: 50)
+        switcher.addTarget(self, action: #selector(changePaymentAction), for: .valueChanged)
+        return switcher
+    }()
+
+    private let cardPaymentLabel = {
+        let label = UILabel()
+        label.text = "Оплата картой"
+        label.font = UIFont.systemFont(ofSize: 24)
+        label.frame = CGRect(x: 100, y: 650, width: 250, height: 30)
+        return label
+    }()
+
+    private let cashPaymentLabel = {
+        let label = UILabel()
+        label.text = "Наличными"
+        label.font = UIFont.systemFont(ofSize: 24)
+        label.frame = CGRect(x: 100, y: 700, width: 250, height: 30)
+        return label
+    }()
+
+    private let orderInfoLabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.frame = CGRect(x: 20, y: 100, width: 300, height: 300)
+        label.text = "Ваш заказ:\n"
+        return label
+    }()
+
+    // MARK: - Open properties
+    
+    var returnHandler: (() -> ())?
+
+    // MARK: - Private properties
+
+    private var fullOrder: [String] = [] {
```

Comment:
```suggestion
    private var fullOrders: [String] = [] {
```
s - множественное число же